### DISCUSSION TITLE
Confused about free trials

### DISCUSSION THREAD

**Discussion by Andrinn** [Glench/ExtPay#105] [Q&A]
Hey!

Is it possible to offer an auto-renewing free trial where users are automatically charged after the trial period ends, without requiring them to enter their information again? This approach could potentially increase the conversion rate of the trial to paid subscriptions.

Currently, some SaaS companies require users to enter their payment details after the free trial period has ended before they can continue using the service. BUT: Most successful SaaS providers, such as Apple's App Store and Google Play Store, have implemented an auto-renewing free trial system where users provide their payment information upfront and are automatically charged after the trial period. This streamlined approach can lead to a higher conversion rate and improved user experience.

I think this would be huge, if possible.

**Comment by burkybang** [comment]
I personally hate when companies do that because then I have to remember when to cancel if I don't want to pay. I hate when I am forced to enter payment details to begin a free trial. That's a huge red flag for me, and I will almost always turn away from a product if it's set up that way.

Yes, it's good for companies because users sometimes forget about it, but I think that's exactly what makes it a bad _user_ experience. I'm certain there are other consumers who feel the same as me, so for that reason, I think a product set up like this will receive less new free trials.

I realize that this is just my opinion, and it's not the intention of this discussion. Regardless of my opinion, having more options is better than not, so I'm in no way against adding this as an optional feature.

**Reply by Andrinn** [reply]
Sure, that's a valid opinion. Personally, I believe the optimal solution would be to provide both options for individuals to choose from. This would allow for a more comprehensive evaluation of the situation and A/B testing.

**Reply by Andrinn** [reply]
Agreed.
